book/src/dev/release-criteria.md

```diff
@@ -0,0 +1,57 @@
+# Go/No-Go Release Criteria
+
+## Alpha Release
+
+In the first alpha release, we want Zebra to participate in the network and replicate the Zcash chain state. We also want to validate proof of work and the transaction merkle tree, and serve blocks to peers.
+
+### System Requirements
+
+_TBD_
+
+### Build Requirements
+
+_TBD_
+
+### Supported Platforms
+
+_TBD_
+
+### Go/No-Go Criteria
+
+- Zebrad Functionality
+    - [ ] zebrad can sync to mainnet tip
+        - [ ] under perfect network conditions (within 2 - 3 hours)
+        - [ ] under reasonable network conditions (in under 5 hours)
+        - [ ] under sub-optimal network conditions (?)
```

Comment:
Let's reduce the number of checklist items here, and the level of quality we're guaranteeing?

```suggestion
    - [ ] zebrad can sync to mainnet tip
        - [ ] under excellent network conditions (within 2 - 5 hours)
        - reasonable and sub-optimal network conditions are not yet supported
```